.github/dependabot.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+updates:
+# Maintain dependencies for each example folder
```

Comment:
You can remove this comment since it applied specifically to the AWS adapter having 4 example folders

Reply:
[Updated](https://github.com/cdktf/cdktf-integration-serverless-python-gcp-example/pull/9/commits/f1520ce1724dbaefbe14034975ce6ca7b2a8d888) 👍